wrappers/nodejs/scripts/npm_dist/build-librealsense.js

```diff
@@ -18,42 +18,37 @@ function buildNativeLib() {
 }
 
 function buildNativeLibLinux() {
-  exec('./scripts/npm_dist/build-dist.sh',
-       (error, stdout, stderr) => {
-    if (error) {
-      console.log('Failed to build librealsense, error:', error);
-      throw error;
-    }
-
-    if (stdout) {
-      process.stdout.write(stdout);
-    }
-
-    if (stderr) {
-      process.stderr.write(stderr);
+  let child = spawn('./scripts/npm_dist/build-dist.sh');
+  child.stderr.on('data', (data) => {
+    // highlight the content with yellow color
+    console.error('\x1b[33m%s\x1b[0m: ', `${data}`);
```

Comment:
Is the color highlight still needed? I think the inspector can do the filter very well.

Reply:
Do you mean let user filter the error output, it might be a bit difficult, we generate a lot of output and console.error didn't have color effect.

Reply:
If you want to make log/warning/error, I'd suggest using mature module like `console-debug`

Reply:
Yes, the color string seems quite strange.

In this use case, the output  we are printing is not controlled by us, it's from the compile output of a child process, and it includes both warning and error. It seems a bit difficult and not necessary to seperate them into warnings and errors. all I just want to do is to highlight it. If we use console-debug's error mechanism, we will mark an original warning as an error, which may seem a bit confused. So I thins maybe we can use 'colors' module, which can be used like:

```
var colors = require('colors');

console.log("The style of this text will be modified".black);
console.log("The style of this text will be modified".red);
```

Reply:
I will use console.error to print messages from stderr, which is simple and generic.